Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -1003,20 +1035,61 @@ protected int GetVoiceCommandIndex(string command)
 
         #endregion VoiceCommands
 
-        #region IMixedRealityTouchHandler
+        #region NearInteractionHandlers
+        /// <summary>
+        /// Is the eventData a grab action or near pointer?
+        /// </summary>
+        /// <param name="eventData"></param>
+        /// <returns></returns>
+        protected bool IsGrab(IMixedRealityPointer pointer, MixedRealityInputAction action)
+        {
+            if (pointer as IMixedRealityNearPointer != null)
+            {
+                return true;
+            }
+
+            return false;
+        }
+
+        protected bool IsControllerGrip(IMixedRealityPointer pointer)
+        {
+            bool hasGrip = false;
+
+            // TODO : detect a controll grip press or grip pose action
```

Comment:
It's generally not great practice in MRTK to have commented out code. Is there a reason you need to keep this here?